docs/sources/_index.md

```diff
@@ -3,6 +3,192 @@ aliases:
+        - enterprise
+        - oss
+
+  - _target: { path: /docs/grafana/*/dashboards/** }
+    labels:
+      products:
+        - cloud
+        - enterprise
+        - oss
+
+  - _target: { path: /docs/grafana/*/datasources/** }
+    labels:
+      products:
+        - cloud
+        - enterprise
+        - oss
+
+  - _target: { path: /docs/grafana/*/explore/** }
+    labels:
+      products:
+        - cloud
+        - enterprise
+        - oss
+
+  - _target: { path: /docs/grafana/*/fundamentals/** }
+    labels:
+      products:
+        - cloud
+        - enterprise
+        - oss
+
+  - _target: { path: /docs/grafana/*/introduction/grafana-cloud/** }
+    labels:
+      products:
+        - cloud
+
+  - _target: { path: /docs/grafana/*/introduction/grafana-enterprise/** }
+    labels:
+      products:
+        - enterprise
+
+  - _target: { path: /docs/grafana/*/panels-visualizations/** }
+    labels:
+      products:
+        - cloud
+        - enterprise
+        - oss
+
+  - _target: { path: /docs/grafana/*/release-notes/** }
+    labels:
+      products:
+        - cloud
+        - enterprise
+        - oss
+
+  - _target: { path: /docs/grafana/*/search/** }
+    labels:
+      products:
+        - cloud
+        - enterprise
+        - oss
+
+  - _target: { path: /docs/grafana/*/setup-grafana/configure-security/audit-grafana/** }
+    labels:
+      products:
+        - cloud
+        - enterprise
+
+  - _target: { path: /docs/grafana/*/setup-grafana/configure-security/configure-authentication/** }
+    labels:
+      products:
+        - cloud
+        - enterprise
+        - oss
+
+  - _target: { path: /docs/grafana/*/setup-grafana/configure-security/configure-authentication/enhanced-ldap/** }
+    labels:
+      products:
+        - cloud
+        - enterprise
+
+  - _target: { path: /docs/grafana/*/setup-grafana/configure-security/configure-authentication/saml/** }
+    labels:
+      products:
+        - cloud
+        - enterprise
+
+  - _target:
+      {
+        path: /docs/grafana/*/setup-grafana/configure-security/configure-database-encryption/encrypt-secrets-using-hashicorp-key-vault/**,
+      }
+    labels:
+      products:
+        - cloud
+        - enterprise
+
+  - _target: { path: /docs/grafana/*/setup-grafana/configure-security/configure-request-security/** }
+    labels:
+      products:
+        - cloud
+        - enterprise
+        - oss
+
+  - _target: { path: /docs/grafana/*/setup-grafana/configure-security/configure-team-sync/** }
+    labels:
+      products:
+        - cloud
+        - enterprise
+
+  - _target: { path: /docs/grafana/*/setup-grafana/configure-security/export-logs/** }
+    labels:
+      products:
+        - cloud
+        - oss
+
+  - _target: { path: /docs/grafana/*/troubleshooting/** }
+    labels:
+      products:
+        - cloud
+        - enterprise
+        - oss
+
+  - _target: { path: /docs/grafana/*/whatsnew/** }
+    labels:
+      products:
+        - cloud
+        - oss
```

Comment:
@chri2547 @jdbaldry - I'm not clear on why this wouldn't apply to Enterprise. Maybe I'm missing something?

Reply:
Per Mitch, we should use all three labels here.

```suggestion
  - _target: { path: /docs/grafana/*/whatsnew/** }
    labels:
      products:
        - cloud
        - enterprise
        - oss
```